### ISSUE TITLE
Moderate to severe

### ISSUE THREAD

**Issue by LM-Ontologies** [obophenotype/human-phenotype-ontology#6488]
**Preferred term label:**
Moderate to severe

**Synonyms**

**Definition (free text, please give PubMed ID)**
Range between moderate and severe

**Parent term (use hpo.jax.org/app)**
https://hpo.jax.org/app/browse/term/HP:0012824

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
I think that it is better not to add X-to-Y terms to the existing severity terms (https://hpo.jax.org/app/browse/term/HP:0012824). Ontology terms are not intended to provide this kind of granularity, and my clinical experience was that people used moderate to severe like terminology because they just did not want to commit  -- for instance with heart murmurs, there are 6 grades that are very well defined, but people still used I-II or II-III more often than anything else, even though actually the definitions allow you to unambiguously say what grade it is.
